src/main/java/ua/softserveinc/tc/util/DateUtil.java

```diff
@@ -41,6 +43,10 @@ public static Date toDate(String date) {
         }
     }
 
+    public static Date toDate(LocalDate date) {
```

Comment:
This is suggesting in every blog. If you have another way to change LocalDate to Date write it please.